src/srcShared/fs.ts

```diff
@@ -60,6 +62,10 @@ export class FileSystemCommon {
         return FileSystemCommon.arrayToString(await this.readFile(path))
     }
 
+    /**
+     * The vscode fs implementation does not explicitly provide an append method
```

Comment:
would be less ambiguous, and less verbose, to mention the exact name of the vscode api?